app/src/androidTest/java/com/karumi/ui/view/MainActivityTest.kt

```diff
@@ -63,24 +64,24 @@ class MainActivityTest : AcceptanceTest<MainActivity>(MainActivity::class.java)
     }
 
     private fun givenThereAreSomeAvengers(numberOfAvengers: Int): List<SuperHero> =
-            givenThereAreSomeSuperHeroes(numberOfAvengers, avengers = true)
+        givenThereAreSomeSuperHeroes(numberOfAvengers, avengers = true)
 
     private fun givenThereAreSomeSuperHeroes(
-            numberOfSuperHeroes: Int = 1,
-            avengers: Boolean = false): List<SuperHero> {
+        numberOfSuperHeroes: Int = 1,
+        avengers: Boolean = false): List<SuperHero> {
         val superHeroes = IntRange(0, numberOfSuperHeroes - 1).map { id ->
             val superHeroName = "SuperHero - " + id
```

Comment:
You can use string interpolation here as well: `val superHeroName = "SuperHero - $id"`, same for the description

Reply:
yes copy and paste error from original code. Sorry